style.css

```diff
@@ -673,6 +516,166 @@ textarea {
     margin-bottom: 1rem;
   }
 
+  .toolbar-ul {
+    display: none;
+  }
+
+  .popup-window {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 16px;
+    gap: 12px;
+    position: absolute;
+    width: 100%;
+    height: 713px;
+    left: calc(50% - 343px / 2);
+    top: 60px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin-left: 0.7rem;
+  }
```

Comment:
- [ ] **[Optional]** Good job on implementing the required changes by the previous Reviewer, However, It would be better to improve the styling of your popup window by making the following changes:
    - In the mobile version, kindly try to align the popup-window to the center of the page to match the chosen Figma design 😉
    - It would be better also to add an overlay background with ```position: fixed``` to prevent scrolling the page while the popup window is active for the best user experience 👍👍